### DISCUSSION TITLE
Getting error when try to store data in android native module(Java)

### DISCUSSION THREAD

**Discussion by chze99** [react-native-async-storage/async-storage#970] [Q&A]
I am new to react native and also android native code. My work require me to store data from native android into AsyncStorage to be retrieve at react native side. I try to follow the example in documentation and end up with error:
cannot find symbol:
AsyncStorageAccess 
Entry

May i know what package should I import/ what variable should I declare to fix this error

**Comment by tido64** [comment]
It looks like you're trying to use an interface that only exists on the Next storage feature. Did you enable it as mentioned in the prerequisites: https://react-native-async-storage.github.io/async-storage/docs/advanced/brownfield#prerequisites
